server/policies.yaml

```diff
@@ -0,0 +1,34 @@
+audience: SLocf7Sa1ibd5GNJMMqO539g7cKvWBOI
+jwtIssuer: https://auth.mozilla.auth0.com/
+policies:
+  - id: "1"
+    description: Allow everyone to read on hello
+    principals:
+      - <.*>
+    actions:
+      - read
+    resources:
+      - hello
+    effect: allow
+  - id: "2"
+    description: Allow everyone to read and create
+    principals:
+      - <.*>
+    actions:
+      - read
+      - create
+    resources:
+      - record
+    effect: allow
+  - id: "3"
+    description: Allow author to update their own record
+    principals:
+      - <.*>
+    actions:
+      - update
+    resources:
+      - record
+    conditions:
+      author:
+        type: InPrincipalsCondition
```

Comment:
I realize this `InPrincipalsCondition` works for a single string. Maybe we should have another one for a list (eg. a list of `collaborators`)